index.html

```diff
@@ -0,0 +1,1570 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
- Use responsive meta tag
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Reply:
done